.jenkins/build-sanity.groovy

```diff
@@ -35,28 +35,55 @@ node('cico-workspace') {
 			}
 			firstAttempt = false
 			cico = sh(
-				script: "cico node get -f value -c hostname -c comment --release=8 --retry-count=${cico_retries} --retry-interval=${cico_retry_interval}",
+				script: "cico node get -f value -c hostname -c comment --release=8 \
+							--retry-count=${cico_retries} --retry-interval=${cico_retry_interval}",
 				returnStdout: true
 			).trim().tokenize(' ')
 			env.CICO_NODE = "${cico[0]}.ci.centos.org"
 			env.CICO_SSID = "${cico[1]}"
+			env.CICO_NODE_SSH = "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE}"
 		}
 	}
 
 	try {
 		stage('prepare bare-metal machine') {
 			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./.jenkins/prepare.sh root@${CICO_NODE}:'
-			sh "ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build/noobaa-core --gitrepo=${ci_git_repo} --ref=${ci_git_ref}"
+			sh "${CICO_NODE_SSH} ./prepare.sh --workdir=/opt/build/noobaa-core --gitrepo=${ci_git_repo} --ref=${ci_git_ref}"
+		}
+
+		stage('stop jobs from the same PR') {
+			def JobName = "noobaa-core_build-sanity"
```

Comment:
just like `currentBuild.number`, there might be a variable for the job name too

Reply:
We can probably do it with `def JobName = System.getenv('JOB_NAME')` but I don't want to allow the system calls.